src/FluffySpoon.AspNet.LetsEncrypt/RenewalFailMode.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace FluffySpoon.AspNet.LetsEncrypt
+{
+	/// <summary>
+	/// Defines the ways errors are handle by the <see cref="ILetsEncryptRenewalService" />.
+	/// </summary>
+	public enum RenewalFailMode
```

Comment:
I love this. Good work! Perhaps we should even set LogAndContinue as the default option?

Reply:
I assumed the current behavior was intended, so I left it as is. But personally I'd prefer the LogAndRetry as default, since I'd want it to expose the issue as fast as possible.

Reply:
The thing about `LogAndRetry` is that it can be dangerous, since it can exhaust the rate limit of LetsEncrypt, leading to throttling for a week or so (I think).

I'm not sure though. I can't decide. I'll let you decide :)

Reply:
That's a good point. LogAndContinue it is :)